### PR TITLE
AppVeyor: Extract package manager versions to variables

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1415)

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -22,7 +22,8 @@ clone_depth: 50
 
 install:
   - git submodule update --init --recursive
-  - npm install -g bower@1.8.8 npm@6.4.0 yarn@1.13.0
+  - npm install -g npm@6.4.0
```
Comment (fviernau) [inline .appveyor.yml]: nit: type in commit message (nevwer)

Comment (sschuberth) [reply .appveyor.yml]: Done.

File: .appveyor.yml [hunk 2]
```diff
@@ -22,7 +22,8 @@ clone_depth: 50
 
 install:
   - git submodule update --init --recursive
```
Comment (fviernau) [inline .appveyor.yml]: commit msg: alread -> already

Comment (mnonnenmacher) [reply .appveyor.yml]: Fixed on behalf of @sschuberth.

**Comment by mnonnenmacher** [comment]
@sschuberth: I have renamed `VIRTUALENV_VERSION` to `VIRTUAL_ENV_VERSION` to work around `PipTest` failing with `TypeError: attribute name must be string, not 'NoneType'`, as there error was caused by the variable, see:
https://gist.github.com/davehunt/5849294#gistcomment-1265367

**Comment by sschuberth** [comment]
Thanks @mnonnenmacher!

**Pull request merged**